alchemist/alchemist-ns3/settings.gradle

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2010-2019, Danilo Pianini and contributors listed in the main project's alchemist/build.gradle file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+project.name = "${artifactId}"
```

Comment:
convert to settings.gradle.kts.

Reply:
I'm having trouble doing this, I can't find an explanation about how to do the same think with Kotlin DSL

Reply:
It's just matter of converting Groovy code to Kotlin
Either copy from the rest of alchemist or paste the problem you're having

Reply:
In the rest of Alchemist only settings.gradle (not kts) is used. I don't know how to set the project name nor how to access the properties defined in gradle.properties from settings.gradle.kts.

Reply:
Remove the file entirely then. Name will be obtained from the containing folder.